package.json

```diff
@@ -59,6 +59,8 @@
     "@babel/preset-typescript": "^7.14.5",
     "@babel/runtime": "^7.14.6",
     "@types/jest": "^27.4.0",
+    "@types/react": "17.0.2",
+    "@types/react-dom": "17.0.2",
```

Comment:
minor concern that these are pinned at version 17 ( and no `^` )but we are still on react 16?  I am assuming that nothing's wrong here but sometimes these things need to be in sync.

Reply:
Pinned per [this stackoverflow response](https://stackoverflow.com/questions/71791347/npm-package-cannot-be-used-as-a-jsx-component-type-errors/71828113#71828113). I was experiencing similar type errors despite making no changes, apparently as a result of dependencies changing. (potentially better to add these in `resolutions` as described?) If I recall correctly, picking a version that started with 16 was giving me different errors. May be worth revisiting regardless, potentially as part of #42.

Reply:
interesting, they also talk about node/npm versions in the discussion there.   If nothing is broken downstream then I'm happy but it looks weird.  I wish we could put comments in json files.

Reply:
worked for me just now to match types version with react version, including ^ - perhaps I got package-lock in a state where it will hold down the correct version?